src/main/java/edu/hillel/homework/lesson26/repository/OrderRepository.java

```diff
@@ -0,0 +1,50 @@
+package edu.hillel.homework.lesson26.repository;
+
+import edu.hillel.homework.lesson26.dao.ProductDAO;
+import edu.hillel.homework.lesson26.model.Order;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class OrderRepository {
```

Comment:
It's a Service layer but not a repository

Reply:
Well done.